contracts/InterchainTokenService.sol

```diff
@@ -313,8 +313,7 @@ contract InterchainTokenService is
     /**
      * @notice Used to deploy an interchain token alongside a TokenManager in another chain.
      * @dev At least the `gasValue` amount of native token must be passed to the function call. `gasValue` exists because this function can be
-     * part of a multicall involving multiple functions that could make remote contract calls. If the `minter` parameter is empty bytes then
-     * a mint/burn TokenManager is used, otherwise a lock/unlock TokenManager is used.
+     * part of a multicall involving multiple functions that could make remote contract calls.
```

Comment:
Explain how the minter is used though. If minter is empty bytes, no additional minter is set on the token, only ITS is allowed to mint. If the token is being deployed on the current chain, `minter` should correspond to an EVM address (as bytes). Otherwise, an encoding appropriate to the destination chain should be used.

Reply:
updated